beesdoo_shift/wizard/subscribe.py

```diff
@@ -115,6 +115,9 @@ def _get_reset_counter_default(self):
     )
     irregular_start_date = fields.Date(string="Start Date", default=fields.Date.today)
     resigning = fields.Boolean(default=False, help="Want to leave the beescoop")
+    create_user = fields.Boolean(
+        defaul=False, help="Create a portal user for the new worker"
```

Comment:
```suggestion
        default=False, help="Create a portal user for the new worker"
```
there is a `t` missing at the end of `default`.